ramls/schema/circulation-rule-request-policy-match.json

```
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Circulation rule match schema",
+  "description": "Describes a matching rule within a set of rules",
+  "type": "object",
+  "properties": {
+    "ruleLine": {
+      "description": "Line number of the matching rule",
+      "type": "integer"
+    },
+    "loanPolicyId": {
```

I think this is meant to be `requestPolicyId`
```suggestion
    "requestPolicyId": {
```

---

Good catch, corrected in https://github.com/folio-org/mod-circulation/pull/190/commits/91f57fec6f3c9ef6886ce4c49014a49fa9da21c9